docs/UserGuide.md

```diff
@@ -59,25 +59,30 @@ Now it's time to **CONQUER** the semester!
 
 ### Prerequisite
 
-> Make sure you have Java `11` installed on your computer by typing `java --version` from your terminal.\
-> If not, please download it from the [Oracle website](https://www.oracle.com/sg/java/technologies/javase/jdk11-archive-downloads.html).
+Make sure you have Java `11` installed on your computer by typing `java --version` from your terminal. The output should be something similar to:
+
+![Java 11](images/java-11.png)
```

Comment:
Perhaps we should include this in text format, i.e. `openjdk 11.0.181...` instead?